eta-company-calendar-frontend/src/app/calendar/components/calendar.component.ts

```diff
@@ -0,0 +1,89 @@
+import { AfterViewInit, Component, OnDestroy, ViewChild } from '@angular/core';
+import { FullCalendarComponent } from '@fullcalendar/angular';
+import { EventInput } from '@fullcalendar/core';
+import enGbLocale from '@fullcalendar/core/locales/en-gb';
+import huLocale from '@fullcalendar/core/locales/hu';
+import dayGridPlugin from '@fullcalendar/daygrid';
+import interactionPlugin from '@fullcalendar/interaction';
+import timeGrigPlugin from '@fullcalendar/timegrid';
+import { TranslateService } from '@ngx-translate/core';
+import { Subject } from 'rxjs';
+import { takeUntil } from 'rxjs/operators';
+
+@Component({
+  selector: 'app-calendar',
+  styles: [`
+    .white-background {
+      background-color: white;
+    }
+    .app-calendar {
+      margin: 0 auto;
+      max-width: 1000px;
+    }
+  `],
+  template: `
+  <div class='app-calendar white-background'>
+    <full-calendar
+      #calendar
+      defaultView="dayGridMonth"
+      [header]="{
+        left: 'prev,next today',
+        center: 'title',
+        right: 'dayGridMonth,timeGridWeek,timeGridDay,listWeek'
+      }"
+      [locales]="locales"
+      [plugins]="calendarPlugins"
+      [events]="calendarEvents"
+      (dateClick)="handleDateClick($event)"
+    ></full-calendar>
+  </div>
+  `
+})
+
+export class CalendarComponent implements AfterViewInit, OnDestroy {
+
+  private destroy$: Subject<boolean> = new Subject<boolean>();
+
+  @ViewChild('calendar') public calendarComponent: FullCalendarComponent;
+
+  public locales: object[] = [enGbLocale, huLocale];
+
+  public calendarPlugins: object[] = [dayGridPlugin, timeGrigPlugin, interactionPlugin];
+
+  private calendarEvents: EventInput[] = [
+    { title: 'Event Now', start: new Date() },
+  ];
+
+  constructor(private readonly translate: TranslateService) {}
+
+  protected handleDateClick(arg: EventInput) {
+    if (confirm('Would you like to add an event to ' + arg.dateStr + ' ?')) {
```

Comment:
ez a string be van égetve? this.translate.instant-el megtudod oldani asszem

Reply:
Ezt a function-t csak azért hagytam benne, hogy ha kész lesz a #28 új értekezlet felvétele, innen is indítható legyen. A beégetett szöveget később törölni fogom belőle.